src/world/message_server.cpp

```diff
@@ -42,13 +44,9 @@ moodycamel::ConcurrentQueue<chat_message_t> outgoing_queue;
 
 std::unique_ptr<SqlConnection>                                        sql;
 std::unordered_map<REGIONALMSGTYPE, std::shared_ptr<IMessageHandler>> regionalMsgHandlers;
-
-typedef std::function<bool(zmq::message_t* data,
-                           zmq::message_t* packet,
-                           zmq::message_t* from,
-                           in_addr         from_addr,
-                           uint16          from_port)>
-    HandleMessageFunc;
+std::unordered_map<uint16, zone_settings_t>                           zoneSettingsMap;
```

Comment:
Can you slap all of these .cpp-specific objects into an anonymous namespace while you're here please?